modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapLink.vue

```diff
@@ -157,9 +164,27 @@ export default {
       this.editor.focus();
     },
     save() {
+      // cleanup incomplete submissions
+      if (this.value.data.target && !this.value.data.href) {
+        delete this.value.data.target;
+      }
       this.editor.commands[this.name](this.value.data);
       this.active = false;
     },
+    keyboardHandler(e) {
+      if (e.keyCode === 27) {
+        this.close();
+      }
+      if (e.keyCode === 13) {
+        if (this.value.data.href || e.metaKey) {
```

Comment:
on `return`:
- if we have an href, consider this a submission
- if we held the meta key, consider this a submission

otherwise, prevent the confusing default behavior of closing the pane without considering new field values